### PR TITLE
Add home page, footer, accessibility improvements, and localizations

### PR DESCRIPTION
**Pull request by jeffstolz**

### CODE REVIEW

File: source/_top-nav.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<nav>
+  <ul class="top-nav">
+    <li class="top-nav__name"><span class="top-nav__name--impact">Jeff</span>Stolz</li>
+    <li class="top-nav__link"><%= link_to t("top-nav.work"), 'index.html' %></li>
+    <li class="top-nav__link"><%= link_to t("top-nav.about"), 'about.html' %></li>
+    <li class="top-nav__link"><%= link_to t("top-nav.contact"), 'contact.html' %></li>
```
Comment (jaclynperrone) [inline source/_top-nav.html.erb]: Similar note to the one above. Change those single quotes to double quotes:
`"index.html"`, etc

File: source/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<footer>
+    <ul class="footer">
+        <li class="footer__link"><%= link_to '860-406-1021', 'tel:#{8604061021}' %></li>
```
Comment (jaclynperrone) [inline source/_footer.html.erb]: I would shy away from making your phone number public. Not entirely sure that's the safest, most conventional way to get in touch with you. I think email and Linkedin would suffice. Curious to hear your thoughts though!

File: source/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+        <li class="footer__link"><%= mail_to 'jstolz123@gmail.com', 'jstolz123@gmail.com' %></li>
```
Comment (jaclynperrone) [inline source/_footer.html.erb]: Consider using an email address obfuscator to prevent robots from scraping your page, grabbing your email address and spamming you. Something like: https://docs.joeworkman.net/general/tips/email-obfuscation (using HTML Encoding)

File: source/_footer.html.erb [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+        <li class="footer__link"><%= link_to 'Linkedin', 'https://www.linkedin.com/in/jeff-stolz-17666725/', target: "_blank" %></li>
```
Comment (jaclynperrone) [inline source/_footer.html.erb]: Both single and double quotes are being used here. We like to stick with double quotes (https://github.com/thoughtbot/guides/tree/master/style/sass). We reserve single quotes for other things like concatenating strings.

File: source/index.html.erb [hunk 5]
```diff
@@ -1,5 +1,18 @@
----
-title: Jeff Stolz | Product Designer
----
-
-
+<main class="index">
+    <h1><%= t("index.hello") %></h1>
+    <h3><%= t("index.welcome") %></h3>
+    <section class="index__cards">
+        <% link_to 'mobile-heartbeat.html' do %>
```
Comment (jaclynperrone) [inline source/index.html.erb]: Single quotes  -> Double quotes

File: source/partials/_top-nav.html.erb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<nav>
+  <ul class="top-nav">
```
Comment (jaclynperrone) [inline source/partials/_top-nav.html.erb]: Consider naming this something that speaks to the purpose of the content, rather than its location. It's more scalable in the long run, since its location could change.

File: source/index.html.erb [hunk 7]
```diff
@@ -1,5 +1,27 @@
...
+    <h3><%= t("index.welcome") %></h3>
+    <section class="index__cards">
+        <% link_to 'mobile-heartbeat.html', :class => "index__case" do %>
+            <div class="index__case--overlay">
+                <p class="index__case-description"><%= t("index.mh-description") %></p>
+            </div>
+            <h4 class="index__case-heading"><%= t("index.mh") %></h4>
+            <%= image_tag 'mh-thumb.png', :class => "index__thumbnail", :alt => t("index.mh-alt") %>
+        <% end %>
+        <% link_to 'hmh-marketplace.html', :class => "index__case" do %>
```
Comment (jaclynperrone) [inline source/index.html.erb]: I notice throughout these files there's a mix of double and single quotes. We like to stick to double quotes (https://github.com/thoughtbot/guides/tree/master/style/sass), so we can reserve single quotes for things like concatenating strings.

File: source/partials/_footer.html.erb [hunk 8]
```diff
@@ -0,0 +1,7 @@
+<footer>
+    <ul class="footer">
+        <li class="footer__link"><%= link_to '860-406-1021', 'tel:#{8604061021}' %></li>
```
Comment (jaclynperrone) [inline source/partials/_footer.html.erb]: I would shy away from providing your phone number. It's not the safest, most conventional way for someone to reach out to you from the internets. I think email/LinkedIn would suffice. Curious to hear your thoughts though!

File: source/partials/_footer.html.erb [hunk 9]
```diff
@@ -0,0 +1,7 @@
...
+        <li class="footer__link"><%= mail_to 'jstolz123@gmail.com', 'jstolz123@gmail.com' %></li>
```
Comment (jaclynperrone) [inline source/partials/_footer.html.erb]: You don't have to do this, but for sharing my personal email, I use an email address obfuscator to prevent robots from scraping my email. Something like this: https://docs.joeworkman.net/general/tips/email-obfuscation (HTML Encoding)

File: source/stylesheets/base/_variables.scss [hunk 10]
```diff
@@ -1,18 +1,29 @@
...
-$breakpoint-medium: 480px;
-$breakpoint-large: 890px;
+$breakpoint-medium: 40em;
+$breakpoint-large: 55em;
 
 // Colors
-$teal: #12939A;
+$teal: #118288;
 $teal-dark: mix($teal, black, 80%);
+$teal-transparent: rgba(17, 130, 136, 0.7);
```
Comment (jaclynperrone) [inline source/stylesheets/base/_variables.scss]: You can use `$teal` here instead of the RGB:
`$teal-transparent: rgba($teal, 0.7);`

File: source/stylesheets/base/_variables.scss [hunk 11]
```diff
@@ -1,18 +1,29 @@
 // Breakpoints
-$breakpoint-medium: 480px;
-$breakpoint-large: 890px;
+$breakpoint-medium: 40em;
+$breakpoint-large: 55em;
 
 // Colors
-$teal: #12939A;
+$teal: #118288;
 $teal-dark: mix($teal, black, 80%);
```
Comment (jaclynperrone) [inline source/stylesheets/base/_variables.scss]: love it!

File: source/stylesheets/base/_variables.scss [hunk 12]
```diff
@@ -1,18 +1,29 @@
...
+$teal-transparent: rgba(17, 130, 136, 0.7);
+$very-light-gray: #F2F2F2;
```
Comment (jaclynperrone) [inline source/stylesheets/base/_variables.scss]: We like to keep the case of the hex codes consistent. According to per https://github.com/thoughtbot/guides/tree/master/style/sass, we can go with either case, but I've found that keeping them all lowercase is easier to maintain and read.

Easy way to do this in Sublime:
Highlight text, and then `cmd + k + l`

File: source/stylesheets/base/_variables.scss [hunk 13]
```diff
@@ -1,18 +1,29 @@
...
 
 // Font Sizes
 $font-size-base: 1rem;
-$font-size-logo: 2rem;
+$font-size-smallest: $font-size-base * 0.6;
+$font-size-small: $font-size-base * 0.75;
+$font-size-medium: $font-size-base * 1.25;
+$font-size-large: $font-size-base * 1.5;
+$font-size-x-large: $font-size-base * 2;
+$font-size-xx-large: $font-size-base * 3;
```
Comment (jaclynperrone) [inline source/stylesheets/base/_variables.scss]: If the larger sizes are getting tshirt sizes, consider using that same convention for the smaller sizes, too, for consistency and scalability. For example:
`$font-size-x-small`

File: source/stylesheets/components/_index.scss [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+    background-color: $teal-transparent;
+    bottom: 0;
+    color: $white;
+    height: 100%;
+    left: 0;
+    opacity: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 100%;
```
Comment (jaclynperrone) [inline source/stylesheets/components/_index.scss]: You don't need this `width` declaration, since you are declaring it expand to `left: 0` and `right: 0`

File: source/stylesheets/components/_index.scss [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.index__case--overlay {
+    background-color: $teal-transparent;
+    bottom: 0;
+    color: $white;
+    height: 100%;
+    left: 0;
+    opacity: 0;
+    position: absolute;
```
Comment (jaclynperrone) [inline source/stylesheets/components/_index.scss]: Bourbon has some nice shorthands you can use. Adding it to your project would be a nice task in a separate PR! (https://www.bourbon.io/)

File: source/stylesheets/base/_layout.scss [hunk 16]
```diff
@@ -1,10 +1,6 @@
-body {
-	font-family: $font-family-base;
-}
-
 ul {
 	list-style: none;
-	padding-inline-start: 0px;
+	padding-inline-start: 0em;
```
Comment (jaclynperrone) [inline source/stylesheets/base/_layout.scss]: Interesting! I've never used this before. In looking at its cross-browser compatibility, IE doesn't support it (big surprise there..). Consider just setting its `padding-left` to 0 take care of compatibility.

File: source/stylesheets/base/_typography.scss [hunk 17]
```diff
@@ -0,0 +1,18 @@
+html {
+    font-family: $font-family-base;
+}
+
+h1 {
+    font-size: $font-size-xx-large;
+    margin-block-end: 0.5rem;
```
Comment (jaclynperrone) [inline source/stylesheets/base/_typography.scss]: Same note as the previous one. This is not supported across all browsers, so consider subbing this with `margin-bottom`

File: locales/en.yml [hunk 18]
```diff
@@ -0,0 +1,15 @@
...
+    work: "Work"
+  index:
+    hello: "Hi, I'm Jeff."
+    hmh: "HMH Marketplace"
+    hmh-alt: "HMH Marketplace case study"
+    mh: "Mobile Heartbeat"
+    mh-alt: "Mobile Heartbeat case study"
+    pega: "Pegasystems"
+    pega-alt: "Pegasystems case study"
+    welcome: "I'm a Product Designer based in Boston."
```
[the remark this answers is not in the document]
Comment (jeffstolz) [reply locales/en.yml]: Thanks! I actually switched to vscode, but this seemed easy enough: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

File: source/stylesheets/base/_variables.scss (hunk 13 above)
Comment (gcstaublin) [reply source/stylesheets/base/_variables.scss]: I'd agree with making the naming convention more consistent.

Another direction you might consider is making the naming math based. If `1rem` is your base, then all names would flow from that `$font-size-two: 2rem;` $font-size-one-and-quarter: 1.25rem`, `$font-size-three-quarters`, etc.

What this fixes is that moment when you realize you need a size between, say `2rem` and `3rem`. Right now, you're stuck and either changing the value of `xx-large` or creating something like `x-larger` or deviating from the convention.

File: source/partials/_footer.html.erb (hunk 8 above)
Comment (jeffstolz) [reply source/partials/_footer.html.erb]: Maybe that's why I've been getting so much spam...

But yeah, good point - I figure I can replace it with Github & Dribble in it's place.

**Comment by gcstaublin** [comment]
It looks like the indentation of your SCSS files might be inconsistent. I'd suggest picking an indentation. My personal preference is to convert tabs to spaces and use 2 spaces for a tab. BUT, not everyone thinks tabs should be spaces. The important point is to be consistent with yourself :)

File: source/stylesheets/base/_variables.scss (hunk 11 above)
Comment (jeffstolz) [reply source/stylesheets/base/_variables.scss]: Thanks - now passes WCAG AA (not AAA though)

**Comment by setphen** [comment]
@jeffstolz The indentation for your `scss` files in this PR looks consistent to me. I checked out other stylesheets in the repo and it does look like there's some mixing of indentation.

For instance the indentation in https://github.com/jeffstolz/portfolio/blob/master/source/stylesheets/base/_layout.scss looks different from https://github.com/jeffstolz/portfolio/blob/master/source/stylesheets/base/_mixins.scss .

If you're not seeing a difference in your text editor, you might enable visual differentiation of tab and space characters to make sure you're not mixing them. 🎿 ☃️

**Comment by jeffstolz** [comment]
Cool, sounds good - I'm going to dig into optimizing my IDE as well. This is my first time with vscode and I need to learn its functionality a bit better.
